send.c

```diff
@@ -1082,13 +1081,14 @@ struct Address *mutt_default_from(void)
     addr = mutt_addr_new();
     if (UseDomain)
     {
-        addr->mailbox =
-            mutt_mem_malloc(mutt_str_strlen(Username) + mutt_str_strlen(fqdn) + 2);
-        sprintf(addr->mailbox, "%s@%s", NONULL(Username), NONULL(fqdn));
+      const char *fqdn = mutt_fqdn(1);
```

Comment:
Since you branched, `mutt_fqdn()` now takes a bool -- this got lost in your merge.